util/chunk/chunk.go

```diff
@@ -133,6 +145,25 @@ func newVarLenColumn(cap int, old *column) *column {
 	}
 }
 
+// RequiredRows returns how many rows is considered full.
+func (c *Chunk) RequiredRows() int {
+	return c.requiredRows
+}
+
+// SetRequiredRows sets the number of required rows.
+func (c *Chunk) SetRequiredRows(requiredRows, maxChunkSize int) *Chunk {
```

Comment:
Why return `*Chunk`, can we remove it?

Reply:
from test-case he want to impl `method chain pattern`

Reply:
Oh, I got it. Thanks